references/search.md

```diff
@@ -39,20 +39,41 @@ curl \
 #### Response: `200 Ok`
 
 ```json
-[
-  {
-    "id": 25684,
-    "title": "American Ninja 5",
-    "poster": "https://image.tmdb.org/t/p/w1280/iuAQVI4mvjI83wnirpD8GVNRVuY.jpg",
-    "overview": "When a scientists daughter is kidnapped, American Ninja, attempts to find her, but this time he teams up with a youngster he has trained in the ways of the ninja.",
-    "release_date": "1993-01-01"
-  },
-  {
-    "id": 25682,
-    "title": "American Ninja 3: Blood Hunt",
-    "poster": "https://image.tmdb.org/t/p/w1280/c7oNrk8bRg0BlmtvidhVD8ivPYT.jpg",
-    "overview": "Jackson is back, and now he has a new partner, karate champion Sean, as they must face a deadly terrorist known as 'The Cobra', who has infected Sean with a virus. Sean and Jackson have no choice but to fight the Cobra and his bands of ninjas.",
-    "release_date": "1989-02-24"
-  }
-]
+{
+  "hits": [
+    {
+      "id": "25684",
+      "title": "American Ninja 5",
+      "poster": "https://image.tmdb.org/t/p/w1280/iuAQVI4mvjI83wnirpD8GVNRVuY.jpg",
+      "overview": "When a scientists daughter is kidnapped, American Ninja, attempts to find her, but this time he teams up with a youngster he has trained in the ways of the ninja.",
+      "release_date": "1993-01-01",
+      "_formatted": {
+        "id": "25684",
+        "title": "American Ninja 5",
+        "poster": "https://image.tmdb.org/t/p/w1280/iuAQVI4mvjI83wnirpD8GVNRVuY.jpg",
+        "overview": "When a scientists daughter is kidnapped, American Ninja, attempts to find her, but this time he teams up with a youngster he has trained in the ways of the ninja.",
+        "release_date": "1993-01-01"
+      }
+    },
+    {
+      "id": "25682",
+      "title": "American Ninja 3: Blood Hunt",
+      "poster": "https://image.tmdb.org/t/p/w1280/c7oNrk8bRg0BlmtvidhVD8ivPYT.jpg",
+      "overview": "Jackson is back, and now he has a new partner, karate champion Sean, as they must face a deadly terrorist known as 'The Cobra', who has infected Sean with a virus. Sean and Jackson have no choice but to fight the Cobra and his bands of ninjas.",
+      "release_date": "1989-02-24",
+      "_formatted": {
+        "id": "25682",
+        "title": "American Ninja 3: Blood Hunt",
+        "poster": "https://image.tmdb.org/t/p/w1280/c7oNrk8bRg0BlmtvidhVD8ivPYT.jpg",
+        "overview": "Jackson is back, and now he has a new partner, karate champion Sean, as they must face a deadly terrorist known as 'The Cobra', who has infected Sean with a virus. Sean and Jackson have no choice but to fight the Cobra and his bands of ninjas.",
+        "release_date": "1989-02-24"
+      }
+    },
+    ...
+  ],
+  "offset": 0,
+  "limit": 20,
+  "processingTimeMs": 32,
+  "query": "american ninja 5"
+}
```

Comment:
Is the `_formatted` necessary to add? This subchild should only be present in the case of a formatting parameter in the query.  It's a bug that should be fixed by meiliSearch. In either case, I can see why you want to report the exact truth in the response example but don't you think this massive response is confusing and not readable?

Reply:
I didn't know for the `_formatted` bug, I will remove it.
For the readable point, I wanted to display the **real** response returns by MeiliSearch and not a kind of one, because I know it confuses me when I read some docs. It gives me the feeling it's not up-to-date, and sometimes, if there are other points unclear, I feel lost. That's why, For me, it's important to display the `hits` and all the other fields.
But if others do not agree with me, I take your side of course :)